tests/fixtures/mqtt.py

```diff
@@ -0,0 +1,98 @@
+# -*- coding: utf-8 -*-
+# (c) 2020 Richard Pobering <richard@hiveeyes.org>
+# (c) 2020 Andreas Motl <andreas@hiveeyes.org>
+# License: GNU General Public License, Version 3
+# Source: https://github.com/hiveeyes/terkin-datalogger/blob/0.13.0/test/fixtures/mqtt.py
```

Comment:
Instead of vendoring this helper code for the second time, we should probably put it into a dedicated package, `pytest-mqtt`.